sections/00_Preamble

```diff
@@ -0,0 +1 @@
+_This chapter is part of the NEO Specification ^[See [NEO Specification](https://github.com/neo-project/specification) repository] initiative, a community-driven technical specification for Neo blockchain._
```

Comment:
What is a "community-driven technical specification"? I think it's just a specification. Please don't add so many attributives.

Reply:
Done.